reana_commons/snakemake.py

```diff
@@ -47,8 +79,45 @@ def snakemake_validate(
         raise REANAValidationError("Snakemake specification is invalid.")
 
 
-def snakemake_load(workflow_file: str, **kwargs: Any) -> Dict:
-    """Load Snakemake workflow specification into an internal representation.
+def snakemake_validate_v8(
+    workflow_file: str, configfiles: List[str], workdir: Optional[str] = None
+):
+    """Seems to be enough for the first validation. We may move to snakemake --dry-run when the validation process will be fully moved to the server-side.
```

Comment:
Snakemake 8 workflow validation function for Python versions >= 3.11.

Note that we may move to using `snakemake --dry-run`  when the validation process will be fully moved to the server side.

_(to make the headline shorter, whilst the comment goes to the docstring body.)_